src/tests/readytorun/JittedMethodsCountingTest/JittedMethodsCountingTest.cs

```diff
@@ -31,13 +33,17 @@ public static int TestEntryPoint()
 
         Console.WriteLine("\nHello World from Jitted Methods Counting Test!");
 
+        // Get the total amount of Jitted Methods.
```

Comment:
This comment is not useful. It is the exact equivalent of the code that follows.